ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx

```diff
@@ -72,19 +72,19 @@ const OperationIcon = ({ operation }: OperationComponentProps) => {
   }
 };
 const OperationAmount = ({ operation }: OperationComponentProps) => {
-  const hasAmount = "amount" in operation;
-  if (!hasAmount) {
+  const hasAccrued = "accrued" in operation;
+  if (!hasAccrued) {
     return null;
   }
```

Comment:
I think we can safely remove this check because already included in the switch defaul.